RedHat-UX/red-hat-design-system
# docs: link card types on Card - Guidelines subpage to their demos or pattern examples

## What I did

**Updated "Patterns" section on Card's "Guidelines" subpage.**

- Changed card grid to tile grid
- Alphabetized card patterns
- Removed pattern descriptions that don't exist on ux.redhat.com or in the RHDS Figma library.

**Removed the following card pattern demos from Card's "Demos" subpage:**

- Asset pattern
- Icon pattern
- Image pattern
- Logo pattern
- Quote pattern
- Title pattern
- Video pattern

**Updated headings on Card pattern page**

Changed any headings that used `<h2>` to `##` for consistent heading styling and to show a copy button.

  _Before:_
![Screenshot 2024-10-29 at 3 34 07 PM](https://github.com/user-attachments/assets/9d257174-b51c-4df0-ba00-32d0ae9a58f3)
  _After:_
![Screenshot 2024-10-29 at 3 36 58 PM](https://github.com/user-attachments/assets/cd809972-9a45-4897-a189-593a7d3c75d7)

## Testing Instructions

Check that the changes above appear in the DP.
- Card's (element) ["Patterns" section](https://deploy-preview-2017--red-hat-design-system.netlify.app/elements/card/guidelines/#patterns) on the "Guidelines" subpage
- [Card's "Demos" subpage](https://deploy-preview-2017--red-hat-design-system.netlify.app/elements/card/demos/)
- [Card pattern examples](https://deploy-preview-2017--red-hat-design-system.netlify.app/patterns/card/examples/)

## Notes to Reviewers
